### ISSUE TITLE
Latest docker image fails with "Fatal error: Loading libsourcekitdInProc.so failed"

### ISSUE THREAD

**Issue by norio-nomura** [realm/SwiftLint#4501]
Reproducing step:
```bash
$ docker run -it --rm ghcr.io/realm/swiftlint:latest swiftlint
SourceKittenFramework/library_wrapper.swift:31: Fatal error: Loading libsourcekitdInProc.so failed
```
It seems broken since 0.50.0-rc.1
```bash
$ docker run -it --rm ghcr.io/realm/swiftlint:0.50.0-rc.1
Unable to find image 'ghcr.io/realm/swiftlint:0.50.0-rc.1' locally
0.50.0-rc.1: Pulling from realm/swiftlint
cf92e523b49e: Already exists 
819ba53d87aa: Pull complete 
a300f2907961: Pull complete 
85e68c2fe7de: Pull complete 
9a8a5824b94d: Pull complete 
f53a82efd4d2: Pull complete 
b42cedafbc47: Pull complete 
4f4fb700ef54: Pull complete 
Digest: sha256:c7c37ff57b315c488b0aca9eaa7b54d6062a98c0b3ea53b5d44878e627b4b868
Status: Downloaded newer image for ghcr.io/realm/swiftlint:0.50.0-rc.1
SourceKittenFramework/library_wrapper.swift:31: Fatal error: Loading libsourcekitdInProc.so failed
```

0.49.1 is ok.
```
$ docker run -it --rm ghcr.io/realm/swiftlint:0.49.1 swiftlint
Linting Swift files in current working directory
Error: No lintable files found at paths: ''
```

**Comment by jpsim** [comment]
Thanks for the report, it's now fixed on `main`. It's nice to see you here, SwiftLint wouldn't be where it is today without your contributions 😄

**Comment by norio-nomura** [comment]
confirmed. Thanks! 🙏🏻
